libfastboot/bootloader.c

```diff
@@ -293,12 +293,12 @@ EFI_STATUS flash_esp(VOID *data, UINTN size)
 
 EFI_STATUS flash_bootloader_a(VOID *data, UINTN size)
 {
```

Comment:
Please provide details why we replace flash_bootloader_verify with flash_partition, since verification is removed.